### ISSUE TITLE
Checkstyle 7.2 Support

### ISSUE THREAD

**Issue by urmuzov** [jshiell/checkstyle-idea#276]
Is there version of this plugin that uses Checkstyle 7.2?

Current version (4.32.2), that uses Checkstyle 7.1.1 failing for our checkstyle.xml with:

```
The Checkstyle rules file could not be loaded.
	cannot initialize module TreeWalker - Property 'ignoredAnnotations' 
	in module DesignForExtension does not exist, 
	please check the documentation
	The file has been blacklisted for 60s.
```

**Comment by jshiell** [comment]
Not at present. I'll try push an updated version this weekend.

**Comment by jshiell** [comment]
Released as 4.33.0.
